packages/just-scripts/src/tasks/esbuildTask.ts

```diff
@@ -0,0 +1,24 @@
+import type { BuildOptions } from 'esbuild';
+import { resolve, TaskFunction } from 'just-task';
+
+export type EsbuildBuildOptions = BuildOptions;
+export interface EsbuildTransformOptions {
+  esbuildOptions: BuildOptions;
+  include: string[] | string;
+}
+
+/**
+ * creates a esbuild task function, checking for esbuild's presence; can be used for bundling or building
+ */
+export function esbuildTask(options: EsbuildBuildOptions = {}): TaskFunction {
+  const esbuildModuleResolution = resolve('esbuild');
```

Comment:
Should this be `resolveCwd` like other tasks?

Reply:
oh yes